db/schema.rb

```diff
@@ -164,6 +164,7 @@
 
   add_index "jobs", ["project_id"], name: "index_jobs_on_project_id", using: :btree
   add_index "jobs", ["status"], name: "index_jobs_on_status", length: {"status"=>191}, using: :btree
+  add_index "jobs", ["user_id"], name: "index_jobs_on_user_id", using: :btree
```

Comment:
When I run the migrations the schema.rb file now shows t.index, but it had a lot of changes, so manually updated the file. 

Should I push the new format schema.rb file?